docs/how-to/fuzzing-log.txt

```diff
@@ -0,0 +1,2927 @@
+﻿.. code-block:: text
+
```

Comment:
This should just be a text file, not a reStructuredText file. We can reference relevant lines with [`literalinclude`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude) and its options.

```suggestion
﻿```